src/index.html

```diff
@@ -7,8 +7,138 @@
+          </ul>
+
+          <div class="page__text">
+            City Apartment is newly renovated in the heart of Budapest, right in the city center.
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Ut enim ad minim veniam.
+          </div>
+          <a href="" class="page__contact-h">Contact host</a>
+          <h3 class="ammenities__title">Amenities</h3>
+          <div class="amenities">
+            <span class="amenities__item">Elevator</span>
+            <span class="amenities__item">Hair dryer</span>
+            <span class="amenities__item">Washer</span>
+            <span class="amenities__item">Kitchen</span>
+          </div>
+          <a href="" class="amenities__show">Show all ammenities</a>
+          <div class="reviews">
+            <h3 class="reviews___heading" >Reviews</h3>
+            <section class="reviews__review">
+              <div class="reviews__review-person " >
+                <span class="reviews__review-person-img  girl-1"></span>
+                <div class="reviews__review-cont">
+                  <div class="reviews__review-person-name">Anastasia</div>
+                  <div class="reviews__review-person-date">September 2016</div>
+                </div>
+              </div>
+              <div class="reviews__review-text">
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                  sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                  Ut enim ad minim veniam.
+                </p>
+              </div>
+            </section>
+            <section class="reviews__review">
+                <div class="reviews__review-person" >
+                  <span class="reviews__review-person-img  girl-2"></span>
+                  <span class="reviews__review-cont">
+                    <div class="reviews__review-person-name">Gaga</div>
+                    <div class="reviews__review-person-date">September 2016</div>
+                  </span>
+                </div>
+                <div class="reviews__review-text">
+                    Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.
+                </div>
+            </section>
+            </div>
+            <h2 class="who-hosted-heading" >Hosted by Jane</h2>
+            <div class="who-hosted-span">Budapest, Hungary</div>
+            <div class="who-hosted-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et magna aliqua.</div>
+            <a href="" class="who-hosted-contact" >Contact host</a>
+        </div>
+        <div class="form" method="GET">
+          <span class="form__price">25$ per night</span>
+          <form action="">
+            <input type="text" class="form__input form__input-date" name = "input-date" placeholder="Date">
```

Comment:
не помещается - переноси
```suggestion
            <input
              type="text"
              class="form__input form__input-date"
              name="input-date"
              placeholder="Date"
            >
```